tdp_server/api/v1/api.py

```diff
@@ -1,8 +1,9 @@
 from fastapi import APIRouter
 
-from tdp_server.api.v1.endpoints import deploy, service
+from tdp_server.api.v1.endpoints import component, deploy, service
 
 api_router = APIRouter()
 
 api_router.include_router(service.router, prefix="/service", tags=["services"])
+api_router.include_router(component.router, prefix="/service", tags=["components"])
```

Comment:
You can not use a prefix like `/service/{service_id}/component` ? It is weird to have `service` and `component` router with the same `prefix` whereas `component` is "inside" a `service`.

Reply:
Done in 03e5a2b